docs/components.md

```diff
@@ -258,8 +258,57 @@ kubectl get route kagenti-ui -n kagenti-system -o jsonpath='{.status.ingress[0].
 
 ## Identity & Auth Bridge
 
+**Repository**: [kagenti/kagenti-extensions/AuthBridge](https://github.com/kagenti/kagenti-extensions/tree/main/AuthBridge)
+
 Kagenti provides a unified framework for identity and authorization in agentic systems, replacing static credentials with dynamic, short-lived tokens. We call this collection of assets **Auth Bridge**.
 
+**Auth Bridge** solves a critical challenge in microservices and agentic architectures: **how can workloads authenticate and communicate securely without pre-provisioned static credentials?**
+
+### Auth Bridge Components
+
+| Component | Purpose | Repository |
+|-----------|---------|------------|
+| **[Client Registration](https://github.com/kagenti/kagenti-extensions/tree/main/AuthBridge/client-registration)** | Automatic OAuth2/OIDC client provisioning using SPIFFE ID | `AuthBridge/client-registration` |
+| **[AuthProxy](https://github.com/kagenti/kagenti-extensions/tree/main/AuthBridge/AuthProxy)** | JWT validation and transparent token exchange | `AuthBridge/AuthProxy` |
+| **SPIRE** | Workload identity and attestation | External |
+| **Keycloak** | Identity provider and access management | External |
```

Comment:
I think it may be helpful to have links for these as resources even if the repos are not owned through kagenti

Reply:
They are just below, line 315 and 323 but I added them here too. Thank you.